cfg.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2014-2019 Yubico AB - See COPYING
+ */
+
+#ifndef CFG_H
+#define CFG_H
+
+#include <stdio.h>
+
+#define CFG_DEFAULT_PATH (SCONFDIR "/pam_u2f.conf")
+#define CFG_MAX_FILE_SIZE 4096 // Arbitrary
```

Comment:
I'd drop the comment

Reply:
When I see a constant, my first thought is "why not double or half that amount"?
Anyway... OK. :grin: no strong opinion.

Reply:
No strong opinions from my side either, I just don't get that much more information from this comment :)